api/src/db/DynamoBusinessesDataClient.ts

```diff
@@ -64,7 +57,7 @@ export const DynamoBusinessesDataClient = (
           return [];
         }
         const businesses = result.Items.map((item) => unmarshall(item));
-        return Promise.all(businesses.map(async (item) => await doMigration(item.data)));
+        return Promise.all(businesses.map(async (item) => await item.data));
```

Comment:
Is this a situation where we want to use `Promise.allSettled` instead of `Promise.all`?